tools/toolchains/iar.py

```diff
@@ -68,20 +80,6 @@ def __init__(self, target, notify=None, macros=None, build_profile=None,
         cxx_flags_cmd = [
             "--c++", "--no_rtti", "--no_exceptions"
         ]
-        if target.core == "Cortex-M7FD":
-            asm_flags_cmd += ["--fpu", "VFPv5"]
-            c_flags_cmd.append("--fpu=VFPv5")
```

Comment:
is anything lost by not specifying the `--fpu` in either the assembler or compiler?

Reply:
In case of IAR no, see https://github.com/ARMmbed/mbed-os/pull/9431#discussion_r249349584